### DISCUSSION TITLE
Couldn't find Odoo.mez file

### DISCUSSION THREAD

**Discussion by Nadalbdh** [tmijail/Odoo-Power-BI-Connector#3] [Q&A] [answered]
Hello, 
Can you please show where the odoo.mez file is ? Couldn't find it for the life of me. 
Thank you.
Edit: Found it in assets :)

**Comment by tmijail** [comment, marked as answer]
As you said, it's in the [releases](https://github.com/tmijail/Odoo-Power-BI-Connector/releases/tag/0.1.0).
I'm commenting just to mark as answered.
